lib/session.ts

```diff
@@ -228,11 +235,29 @@ export class Session {
   }
 
   private initializeOps(graph: Graph): void {
+    graph.partitionBy(this.supportedOps);
```

Comment:
Do you consider to encapsulate the graph manipulation into `sessionHandler.transformGraph` of WebNN backend?

Reply:
I agree. I'll make it more consistent with their original design later.